src/resource/image.rs

```diff
@@ -210,26 +213,71 @@ impl<A: Allocator> Image<A> {
     }
 
     /// Construct a trivial [`ImageView`] from this [`Image`]. This is an image view that views the
-    /// entire image subresource.
+    /// whole image subresource.
+    /// * `aspect` - The image aspect flags that will be used to create the image view
     /// <br>
     /// <br>
     /// # Lifetime
     /// The returned [`ImageView`] is valid as long as `self` is valid.
-    pub fn view(&self, aspect: vk::ImageAspectFlags) -> Result<ImageView> {
+    pub fn whole_view(
+        &self,
+        aspect: vk::ImageAspectFlags,
+        view_type: vk::ImageViewType,
+    ) -> Result<ImageView> {
+        // Node from nodle man: I don't think this should even be a function tbh because
+        // the view() function itself isn't *that* bloated with arguments 
+        self.view(
+            aspect,
+            view_type,
+            0,
+            None,
+            0,
+            None
+        )
+    }
+
+    /// Construct a trivial [`ImageView`] from this [`Image`]. This is an image view that views the
```

Comment:
```suggestion
    /// Construct an [`ImageView`] from this [`Image`]. This is an image view that views the
```
minor update in docs